torch_em/util/prediction.py

```diff
@@ -136,6 +136,7 @@ def predict_with_halo(
     tqdm_desc: str = "predict with halo",
     prediction_function: Optional[Callable] = None,
     roi: Optional[Tuple[slice]] = None,
+    iter_list: Optional[List[int]] = [],
```

Comment:
A minor comment: `Optional` means that the value can be `None` and typically the default value should then also be `None` (the usual pattern in python).